code/src/Albums.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Artist } from './Artist';
+import { ReactComponent as PlayButton } from './play.svg';
+import { ReactComponent as HeartButton } from './heart.svg';
+import { ReactComponent as DotButton } from './dots.svg';
+
+export const Albums = (props) => {
+	if (props.type === 'album') {
+		return (
+			<>
```

Comment:
Woooow, some sick indentation :D
Please, for future projects try to pay attention to that :)

Moreover, if your component returns only one root element, there is no need to wrap it around React Fragment.

In this case WE NEED to use React Fragment :

```jsx
return (
    <div>My element 1</div>
    <div>My element 2</div>
);
```

so, after necessary changes it looks like this :

```jsx
return (
    <>
        <div>My element 1</div>
        <div>My element 2</div>
    </>
);
```

But in this case WE DON'T NEED to use React Fragment :

```jsx
return (
    <div>My element 1</div>
);
```